protoc-gen-hack/plugin.go

```diff
@@ -172,6 +174,23 @@ func toPhpName(ns, name string) (string, string) {
 	return strings.Replace(ns, ".", "\\", -1), strings.Replace(name, ".", "_", -1)
 }
 
+func isReservedClassName(name string) bool {
```

Comment:
Also: lets rename this isReservedName, and similarly escapeReservedName below. We should do this for class members and enum names and all the other parts, eventually (not asking you to do this now unless you want to).

Reply:
Done.